src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -47,7 +50,10 @@
                 <attribute name="messenger">input</attribute>
                 <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\PickupCart</attribute>
                 <attribute name="denormalization_context">
-                    <attribute name="groups">shop:order:create</attribute>
+                    <attribute name="groups">
+                        <attribute>shop:order:create</attribute>
+                        <attribute>sylius:shop:order:create</attribute>
+                    </attribute>
                 </attribute>
                 <attribute name="normalization_context">
                     <attribute name="groups">
```

Comment:
These groups are missing prefixes, and it's the same if four other places in this file